chapters/vulkan_profiles.adoc

```diff
@@ -0,0 +1,504 @@
+
+The real purpose of the LunarG Desktop Baseline profiles is to provide examples to Vulkan application developers to create profiles for their applications. It's important to note that these profiles have some limitations - for example, Windows ARM / Qualcomm GPUs are not supported because these GPUs were not released or just released when the profiles were generated.
+
+If LunarG were to create a Desktop Baseline 2025 in the future, it would likely include Windows ARM / Qualcomm GPUs, which would make the Desktop Baseline 2025 both a subset and superset (likely Vulkan 1.3) of the Desktop Baseline 2024.
+
+The profile includes:
+
+* Core Vulkan 1.1 functionality
+* Key extensions such as:
+** `VK_KHR_maintenance1`, `VK_KHR_maintenance2`, `VK_KHR_maintenance3`
+** `VK_KHR_create_renderpass2`
+** `VK_KHR_image_format_list`
+** `VK_KHR_sampler_mirror_clamp_to_edge`
+* Specific feature requirements include:
+** `robustBufferAccess`
+** `fullDrawIndexUint32`
+** `imageCubeArray`
+** `independentBlend`
+** And many more core features
+
+This profile ensures that applications focusing on it will work consistently across a wide range of desktop GPUs from different vendors.
+
+=== Android Baseline 2022 Profile Details
+
+The Android Baseline 2022 profile (`VP_ANDROID_BASELINE_2022`) is tailored specifically for Android devices and includes:
+
+* Core Vulkan 1.1 functionality
+* Android-specific extensions such as:
+** `VK_KHR_android_surface`
+** `VK_ANDROID_external_memory_android_hardware_buffer`
+* Common rendering extensions:
+** `VK_KHR_swapchain`
+** `VK_KHR_maintenance1`, `VK_KHR_maintenance2`, `VK_KHR_maintenance3`
+** `VK_KHR_dedicated_allocation`
+* Mobile-optimized limits and features
+
+This profile is particularly valuable for Android game developers who want to ensure their applications run on a wide range of Android devices without having to implement complex device-specific code paths.
+
+=== Roadmap 2022 Profile
+
+The Roadmap 2022 profile (`VP_KHR_ROADMAP_2022`) is a forward-looking profile that defines capabilities that hardware vendors are encouraged to support in future implementations. It includes:
+
+* More advanced features and extensions beyond the baseline profiles
+* Higher minimum limits for various resources
+* Support for newer Vulkan functionality
+
+By targeting this profile, developers can prepare their applications for future hardware capabilities while still maintaining compatibility with current devices through fallback mechanisms.
+
+=== Profile Versioning Strategy
+
+Profiles use a year-based versioning scheme (e.g., 2022, 2023) to indicate when they were defined. This approach:
+
+* Provides clear temporal context for each profile
+* Allows for the creation of new profiles that reflect evolving hardware capabilities
+* Maintains backward compatibility, as older profiles remain valid even as
```

Comment:
The second part somewhat feels like a contradiction (to the first part). Mayb rephrase to "are updated to fix shortcomings or issues"